test/cmocka/src/audio/buffer/buffer_wrap.c

```diff
@@ -33,7 +33,13 @@ static void test_audio_buffer_write_fill_10_bytes_and_write_5(void **state)
 
 	uint8_t bytes[10] = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
 
```

Comment:
I feel that prefix for the commit subject should be ``test: buffer: `` instead of Buffer UT``

Reply:
changed